tools/docs/config.ts

```diff
@@ -1,11 +1,28 @@
-import table from 'markdown-table';
 import { getOptions } from '../../lib/config/options';
 import { getCliName } from '../../lib/workers/global/config/parse/cli';
 import { getEnvName } from '../../lib/workers/global/config/parse/env';
 import { readFile, updateFile } from '../utils';
 
 const options = getOptions();
 
+function buildHtmlTable(data: string[][]): string {
+  const ind = '  '; // indentation 2 spaces
+  let table = `<table>\n`;
+  for (const [i, row] of data.entries()) {
+    table += `${ind}<tr>\n`;
```

Comment:
I believe that the first table row should be wrapper in a `thead` element, and the rest should be contained by a `tbody` element. That's how the current documentation is rendered anyway.